wiki/Tournaments/SSR/1/en.md

```diff
@@ -0,0 +1,955 @@
+   - Player point-of-view live stream snippets (commonly referred to as "clips" or "VODs"). The entirety of the play, along with the results screen must be clearly visible along with the affected player's score.
+   - Replay files of the play, taken directly from the "Local scores" tab on the affected player's client (the timestamps must exactly match the time at which the game took place, as seen on the multiplayer lobby link).
+   - Player scores may be derived from the official stream as a last resort, in cases where the match is streamed.
+10. Players are expected to keep the match running fluently and without delays. Excessive match delays from the players' side may result in penalties being applied by the tournament organisers. Disrupting the match by foul play, insulting or provoking other players or staff, delaying the match, and other deliberate inappropriate misbehaviour are strictly prohibited, and will be punished accordingly.
+11. Any attempts to manipulate bracket outcomes or future matchups may result in the immediate disqualification of the player from the competition. Examples of manipulative behavior include, but are not limited to:
+    - Deliberately playing poorly to lose a game or match.
+    - Colluding with other players to force a desired matchup.
+    - Forfeiting bracket matches without providing an adequate reasoning, or with the intention of getting a more favourable matchup.
+12. All players and staff must be treated with respect. Instructions from the referees and the tournament organisers are to be followed. Decisions labelled as final are not to be objected.
+13. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms where the matches will take place.
+    - Breaking the chat rules may result in a silence. Silenced players cannot participate in multiplayer matches for the duration of the punishment and, as a result, can have their match forfeited.
+14. Penalties for violating the tournament rules include, but are not limited to:
+    - Removal of pick rights for a map or an entire match.
+    - Point deductions.
+    - Declaring the match as forfeited, or as a win by default for the other player.
+    - Disqualification from the entire tournament.
+    - Disqualification from the current and future official tournaments, until appealed.
+15. Referees may allow, at their discretion, lower or higher tolerances for timers.
+16. The tournament organisers may request liveplays or recordings of individual players at any point in the tournament without prior warning.
+17. The tournament organisers will be responsible for receiving and investigating any tournament-related complaints.
+18. The tournament organisers reserve the right to modify these rules at any moment. Any such changes will be announced in advance.
+
+### Registration
+
+1. Every user interested in playing will be be required to sign up through the Player Registrations form.
```

Comment:
```suggestion
1. Every user interested in playing will be be required to sign up through the player registrations form.
```

Also no big deal, but the link to the form is not present in the article. Kinda whatever as the tournament is, y'know, over. But just in case you want to add it under the "Links" section.